src/components/interaction-designer/flow-editors/import/TechErrorNotifications.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="tech-error-notification">
+    <div v-for="(ajvError, key) in validationErrors" :key="key">
+      <h6>
+        <b>#{{ key + 1 }} &nbsp;</b>
+        <b>Location</b>: {{ ajvError.dataPath }}, <b>Message</b>: {{ ajvError.message }}
```

Comment:
Should we consider translating these strings?

Reply:
Good catch